gobblin-api/src/main/java/gobblin/util/RecordCountProvider.java

```diff
@@ -37,7 +37,7 @@
    * Convert a {@link Path} from another {@link RecordCountProvider} so that it can be used
    * in {@link #getRecordCount(Path)} of this {@link RecordCountProvider}.
    */
-  public Path convertPath(Path path, RecordCountProvider src) {
+  public Path convertPath(Path path, String extension, RecordCountProvider src) {
```

Comment:
It's used in MRCompactorJobRunner as .avro was hardwired.
https://github.com/linkedin/gobblin/pull/1686/commits/f8fc1d890f74ef6329d0b5c12802d5136833354a#diff-dfd13665b702c59b24278904778cb5cdL326